test/db_root_command_shell_test.go

```diff
@@ -271,6 +271,19 @@ func (s *DBRootCommandShellSuite) Test_GivenATableNameWithSpecialCharacters_When
 	s.tc.AssertSqlEquals(outS, expected)
 }
 
+func (s *DBRootCommandShellSuite) Test_GivenATableNameWithTheSameSignatureAsExpainQueryPlan_WhenQueryingIt_ExpectNotToBeTreatedAsExplainQueryPlan() {
+	_, _, err := s.tc.Execute("CREATE TABLE fake_explain (ID INTEGER PRIMARY KEY, PARENT INTEGER, NOTUSED INTEGER, DETAIL TEXT);")
+	s.tc.Assert(err, qt.IsNil)
+
+	outS, errS, err := s.tc.ExecuteShell([]string{"SELECT * FROM fake_explain;"})
```

Comment:
shouldn't you be testing the `EXPLAI QUERY PLAN` function?

Reply:
Actually this is testing that a table with the same column names as the EQP table will still be treated as a normal table (which it should).